data/projects.js

```diff
@@ -459,5 +459,25 @@ coala should support generating metrics for your code.",
 		"tags" : ["Documentation", "Bears"],
 		"markdown": "documentation_extraction.md",
 		"collaborating_projects" : ["coala"]
+	},
+	{
+		"name" : "Integrate pyflakes-enhanced AST into coala ",
+		"desc" : "Integrate pyflakes-enhanced AST into coala \
+		so that bears can access it.",
+		"requirements" : [
+                                 "The participant has a patch to any bear merged. Check out \
+                                 [existing bear proposal](https://github.com/coa\
```

Comment:
this is a list of new bear proposals. I think as a requirement it is enough to also just fix a bug.